setup.py

```diff
@@ -0,0 +1,465 @@
+    return old_functions
+
+
+def combine_constants(old_constants: Dict[str, str], new_constants: Dict[str, str]) -> Dict[str, str]:
+    for key, value in new_constants.items():
+        old_constants[key] = value
+    return old_constants
+
+
+ignored_dependencies = [
+    'bit', 'boolean', 'Vector', 'List', 'Container', 'BLSPubkey', 'BLSSignature',
+    'Bytes1', 'Bytes4', 'Bytes32', 'Bytes48', 'Bytes96', 'Bitlist', 'Bitvector',
+    'uint8', 'uint16', 'uint32', 'uint64', 'uint128', 'uint256',
+    'bytes', 'byte', 'ByteList', 'ByteVector'
+]
+
+
+def dependency_order_ssz_objects(objects: Dict[str, str], custom_types: Dict[str, str]) -> None:
+    """
+    Determines which SSZ Object is dependent on which other and orders them appropriately
+    """
+    items = list(objects.items())
+    for key, value in items:
+        dependencies = []
+        for line in value.split('\n'):
+            if not re.match(r'\s+\w+: .+', line):
+                continue  # skip whitespace etc.
+            line = line[line.index(':') + 1:]  # strip of field name
+            if '#' in line:
+                line = line[:line.index('#')]  # strip of comment
+            dependencies.extend(re.findall(r'(\w+)', line))  # catch all legible words, potential dependencies
+        dependencies = filter(lambda x: '_' not in x and x.upper() != x, dependencies)  # filter out constants
+        dependencies = filter(lambda x: x not in ignored_dependencies, dependencies)
+        dependencies = filter(lambda x: x not in custom_types, dependencies)
+        for dep in dependencies:
+            key_list = list(objects.keys())
+            for item in [dep, key] + key_list[key_list.index(dep)+1:]:
+                objects[item] = objects.pop(item)
+
+
+def combine_ssz_objects(old_objects: Dict[str, str], new_objects: Dict[str, str], custom_types) -> Dict[str, str]:
+    """
+    Takes in old spec and new spec ssz objects, combines them,
+    and returns the newer versions of the objects in dependency order.
+    """
+    for key, value in new_objects.items():
+        old_objects[key] = value
+    return old_objects
+
+
+def combine_spec_objects(spec0: SpecObject, spec1: SpecObject) -> SpecObject:
+    """
+    Takes in two spec variants (as tuples of their objects) and combines them using the appropriate combiner function.
+    """
+    functions0, custom_types0, constants0, ssz_objects0 = spec0
+    functions1, custom_types1, constants1, ssz_objects1 = spec1
+    functions = combine_functions(functions0, functions1)
+    custom_types = combine_constants(custom_types0, custom_types1)
+    constants = combine_constants(constants0, constants1)
+    ssz_objects = combine_ssz_objects(ssz_objects0, ssz_objects1, custom_types)
+    return SpecObject(functions, custom_types, constants, ssz_objects)
+
+
+def dependency_order_spec(objs: SpecObject):
+    functions, custom_types, constants, ssz_objects = objs
+    dependency_order_ssz_objects(ssz_objects, custom_types)
```

Comment:
Could be clearer:
```python
def dependency_order_spec(spec_obj: SpecObject): 
    dependency_order_ssz_objects(spec_obj.ssz_objects, spec_obj.custom_types)
```

Reply:
Removing the function altogether, there's only 1 caller.